libraries/inverse/dipoleFit/dipole_fit_data.cpp

```diff
@@ -605,7 +605,9 @@ void mne_free_cov_3(MneCovMatrix* c)
     if (c->sss)
         delete c->sss;
     c->bads.clear();
-    FREE_3(c);
```

Comment:
Are we sure this other matrix that was changed from Free3() to delete uses new? The source code in mne_cov_matrix still uses a lot of malloc. Is this another coverity identified issue?